docs/_book.yaml

```
@@ -114,12 +114,22 @@ upper_tabs:
 
     - name: "Tutorials"
       contents:
-        ####  CIRQ ####
-      - heading: "Cirq"
+      ####  OVERVIEW  ####
+      - title: "Cirq tutorials and examples"
+        path: /cirq/tutorials/examples
```

Let's rename this to index

```suggestion
        path: /cirq/tutorials/index
```